File: SmartBrowser_java_eclipse/src/com/teuskim/sbrowser/InstructionsActivity.java
package com.teuskim.sbrowser;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;

public class InstructionsActivity extends Activity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.instructions);
		findViews();
	}
	
	private void findViews(){
		findViewById(R.id.btn_close_instructions).setOnClickListener(new OnClickListener() {
			
			@Override
			public void onClick(View v) {
				finish();
			}
		});
	}

}

File: SmartBrowser_kotlin/app/src/main/java/com/teuskim/sbrowser/InstructionsActivity.kt
package com.teuskim.sbrowser

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.view.View.OnClickListener

class InstructionsActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.instructions)
        findViews()
    }

    private fun findViews() {
        findViewById<View>(R.id.btn_close_instructions).setOnClickListener { finish() }
    }

}
